x/expiration/keeper/keeper.go

```diff
@@ -95,31 +95,31 @@ func (k Keeper) GetModuleAccount(ctx sdk.Context) authtypes.ModuleAccountI {
 	return k.acctKeeper.GetModuleAccount(ctx, types.ModuleName)
 }
 
-// GetDeposit returns the default deposit used in setting module asset expirations
-func (k Keeper) GetDeposit(ctx sdk.Context) sdk.Coin {
+// GetDefaultDeposit returns the default deposit used in setting module asset expirations
+func (k Keeper) GetDefaultDeposit(ctx sdk.Context) sdk.Coin {
```

Comment:
I'm sorry. I now think I was wrong about this change. It was better named as `GetDeposit` since it looks up the deposit from the params (and doesn't just return a default). However, the word "default" should be removed from the comment.